### ISSUE TITLE
How to make inference the model

### ISSUE THREAD

**Issue by amgomezdev** [huggingface/autotrain-advanced#225]
When I launch 
**autotrain llm --train --project_name my-llm --model meta-llama/Llama-2-7b-hf --data_path . --use_peft --use_int4 --learning_rate 2e-4 --train_batch_size 2 --num_train_epochs 3 --trainer sft**

I have this output
![autoTrainDoubt](https://github.com/huggingface/autotrain-advanced/assets/30750249/ac813d13-d4a4-43f4-901a-372fdaec045b)

**I have two questions.**
**1.-** The output is telling that the training is finished, however I only watch the log of 1 epoch. **Is there any way to see the 'training loss' param of the 3 epochs**.

**2.-** After training, I try to make inferece with Text-generation-Inference HF application. However I have an error because config.json is not in the model folder. The output model is this. **Why is not present this file? Should I do something more?**.

![autoTrainDoubt1](https://github.com/huggingface/autotrain-advanced/assets/30750249/2af0d7fe-526c-4646-aa64-9adf6d70632f)

**Comment by msobo** [comment]
add "--merge-adapter" to your autotrain and train again.
or 
https://github.com/huggingface/autotrain-advanced/issues/211#issuecomment-1681826693

**Comment by abhishekkrthakur** [comment]
you can also merge without needing to train again

**Comment by TapendraBaduwal** [comment]
@abhishekkrthakur sir After merging my the model, it generates text learned from both the base model and my new training datasets. How can I make the model which make answer  on my custom training datasets only?

**Comment by amgomezdev** [comment]
@abhishekkrthakur thanks. I have merged the model but when I inference the new model, I have two type of answers.
Base model answer and trained model answer.
Is there any solution to have only trained model answer?
